src/pilots.js

```diff
@@ -5006,7 +5076,7 @@ window.PILOTS = [
 	{
             name: "Thweek",
             faction:Unit.SCUM,
-            done:false,
+            done:true,
```

Comment:
For Thweek we still need to implement the Mimic condition, and the selection of which condition he wants to use at the beginning..

Reply:
Ah, damn.  Sorry, I thought that was already in.